app/assets/javascripts/search-result-load-in.js

```diff
@@ -3,12 +3,12 @@ $(document).ready(function() {
   $('.search-results__card').each(function() {
     $(this)
       .css({
-        transitionDelay: (searchResult * 0.05) + 's',
+        transitionDelay: (searchResult * 0.10) + 's',
       })
       .addClass('show-card');
-      setTimeout(function() {
-        $('.search-results__card').removeClass('search-results--placeholder');
-      }, searchResult * 50 );
     searchResult++;
   });
+  $(window).on('load', function() {
+    $('.search-results__card').removeClass('search-results--placeholder');
+  });
```

Comment:
One small thing here. Dan had me wrap my JavaScript in an if statement that checks if there are elements to assign these sort of things on before assigning them. So what that would look like is something like...
```
$(document).ready(function() {
  if ($('.search-results__card').length > 0) {
    // the .each() method
    // the window load check
  }
});
```
JavaScript does some bad things with memory if you have methods run on things that don't exist. This will also help make sure all the JavaScript loads faster on pages that _don't_ include this class.